packages/itmat-ui-react/src/components/datasetDetail/tabContent/dashboard/jobs.tsx

```diff
@@ -30,7 +30,7 @@ const JOBTYPES: { [type: string]: any } = {
     FIELD_INFO_UPLOAD: <span>Field annotation upload</span>
 };
 
-export const JobSection: React.FunctionComponent<{ studyId: string, jobs: Array<IJobEntry<any>> }> = ({ studyId, jobs }) => {
+export const JobSection: React.FunctionComponent<{ studyId: string; jobs: Array<IJobEntry<any>> }> = ({ studyId, jobs }) => {
```

Comment:
why is the interface key separator ";" in ui but "," in the other packages? maybe we should do comma all the way